### ISSUE TITLE
2.12 regression (?): macro-paradise started failing in the community build

### ISSUE THREAD

**Issue by SethTisue** [scala/scala-dev#571]
after scala/community-builds@ae1d4e8b41baa1c7cf9e33e2ff346b80b3686415, at e.g. https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-integrate-community-build/3717/ , we see

```
[macro-paradise] java.lang.AssertionError: assertion failed: 
[macro-paradise]   (class InteropIdentity,copy$default$1)
[macro-paradise]      while compiling: /home/jenkins/workspace/scala-2.12.x-integrate-community-build/target-0.9.16/project-builds/macro-paradise-d915ab9c3ca61a6d3b3098d0804f2bda3d128da8/tests/src/test/scala/annotations/run/InteropCaseSynthesis.scala
[macro-paradise]         during phase: typer
[macro-paradise]      library version: version 2.12.8-bin-0f07cc1
[macro-paradise]     compiler version: version 2.12.8-bin-0f07cc1
```

attn @adriaanm 

the earliest I could bisect it is probably Wednesday or Thursday

**Comment by retronym** [comment]
Sounds like: https://github.com/scala/scala/pull/7203

https://github.com/scalacommunitybuild/paradise/tree/community-build-2.12 needs to be updated with "enter default getters eagerly" from https://github.com/scalamacros/paradise/commits/2.12.8

**Comment by adriaanm** [comment]
I verified my 2.12.8 branch fixes that problem a while back, but it looks like the two branches have diverged? https://github.com/scalacommunitybuild/paradise/compare/community-build-2.12...adriaanm:2.12.8?expand=1

**Comment by SethTisue** [comment]
I just merged your branch onto the community-build-2.12 branch, presumably that'll fix it in the next run. I'll check on it.

**Comment by SethTisue** [comment]
green since https://scala-ci.typesafe.com/view/scala-2.12.x/job/scala-2.12.x-integrate-community-build/3722/
